### ISSUE TITLE
devector

### ISSUE THREAD

**Issue by Aurora11111** [philipperemy/deep-speaker#44]
if we change the embeddings produced by softmax into dvector embeddings which created by pytorch-speaker-verification. can it works?

**Comment by philipperemy** [comment]
Yes it should work.
